src/component/component.yaml

```diff
@@ -0,0 +1,28 @@
+id: PCLCrypto
+name: Cryptography Plugin
+publisher: Andrew Arnott
+publisher-url: https://twitter.com/aarnott
+summary: PCL Storage provides a consistent, portable set of crypto APIs across .NET platforms.
```

Comment:
I found and fixed the source of this "PCL Storage" reference in 4ca0e3ec5. I've also committed a fix for this copy of the mistake in a local commit.